src/test/java/com/znsio/sample/e2e/screen/web/ZomatoResturantPageScreenWeb.java

```diff
@@ -0,0 +1,45 @@
+package com.znsio.sample.e2e.screen.web;
+
+import com.context.TestExecutionContext;
+import com.znsio.e2e.runner.Runner;
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.screen.zomato.ZomatoResturantPageScreen;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebDriver;
```

Comment:
do we need to import selenium webdriver class

Reply:
yes @vishal07111998  -I  have used WebElement to store the data in list format from dropdown.

and also for comapring it with provided text

<img width="867" alt="Screenshot 2022-11-21 at 5 37 12 PM" src="https://user-images.githubusercontent.com/117148030/203050101-267c0f0e-b069-45c5-9c93-32e2de9f513e.png">